app/Home/Home.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  width: full;
```

Comment:
bug: `width: full` is not valid html. What are you trying to do here?

Reply:
FIxed now, I used too much Tailwildcss3